### ISSUE TITLE
[Bug/Question] 1.5.2 Concurrent Helm Chart Race Condition

### ISSUE THREAD

**Issue by SizZiKe** [kubernetes-sigs/aws-load-balancer-controller#3170]
**Describe the bug**
In the 1.5.2 version of the helm chart, a `MutatingWebhook` was introduced that temporarily caused concurrent helm deployments to fail. We seed the cluster with multiple helm charts via Terraform (e.g. efs-csi-driver, external-dns, etc.) and have done this without issue prior to 1.5.2. It seems that there is a race condition of the load balancer controller pods being ready to service the webhook as other pods are being deployed.

My question is whether or not there should be a value in values.yml exposed to allow other `kubernetes.io/name` to be ignored, so that this is possible, or if this is bad practice and the charts should be deployed sequentially. If the charts are deployed sequentially, there is no issue.

**Steps to reproduce**
Deploy several helm charts concurrently with the aws-load-balancer-controller helm chart.

**Expected outcome**
Successful deployment

**Environment**

* AWS Load Balancer controller version: 1.5.2
* Kubernetes version: 1.26
* Using EKS (yes/no), if so version? 1.26

**Additional Context**:
```log
│ Error: Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-webhook-service.kube-system.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-webhook-service"
│ 
│   with module.eks.module.external_dns[0].module.external_dns.helm_release.this[0],
│   on .terraform/modules/eks.external_dns.external_dns/helm.tf line 1, in resource "helm_release" "this":
│    1: resource "helm_release" "this" {
│ 
```

**Comment by M00nF1sh** [comment]
@SizZiKe 
You should install the charts sequentially.
Once Kubernetes 1.27 is generically available, we can further scope our Service webhook configuration to only match against Service with LoadBalancer type. 
For now, you can also set  enableServiceMutatorWebhook to `false` to disable this webhook: https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/helm/aws-load-balancer-controller/values.yaml#L353

**Comment by michemache** [comment]
@M00nF1sh I also run into that race condition. Do you plan to fix this, as EKS 1.27 is available?

**Comment by Julian-Chu** [comment]
We  add dependency in terraform module,  it mitigates the issue, but still happens sometime
